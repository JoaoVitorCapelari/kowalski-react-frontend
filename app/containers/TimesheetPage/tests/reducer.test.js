
import { fromJS } from 'immutable';
import timesheetPageReducer from '../reducer';

describe('timesheetPageReducer', () => {
  it('returns the initial state', () => {
    expect(timesheetPageReducer(undefined, {})).toEqual(fromJS({}));
  });
});
