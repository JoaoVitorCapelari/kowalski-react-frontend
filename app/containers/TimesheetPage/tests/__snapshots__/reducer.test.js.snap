// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timesheetPageReducer returns the initial state 1`] = `
Immutable.Map {
  isTaskDialogOpen: false,
  isDeletingRecord: false,
  isLoadingTimeRecords: false,
  isTaskOverlaySelectOpened: false,
  deleteTimeRecordConfirmDialogOpened: false,
  cachedSelectedDate: null,
  formActivities: Immutable.List [
  ],
  loadingTimeRecordsErrorMsg: "",
  timeRecords: Immutable.List [
  ],
  activityIdLoadedIntoTaskDialog: null,
  formProjects: Immutable.List [
  ],
}
`;
